aleth-vm/main.cpp

```diff
@@ -259,13 +259,18 @@ int main(int argc, char** argv)
         }  // Ignore decoding errors.
     }
 
+    unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(networkName)).createSealEngine());
+    LastBlockHashes lastBlockHashes;
+    EnvInfo const envInfo(blockHeader, lastBlockHashes, 0);
+
     Transaction t;
     Address contractDestination("1122334455667788991011121314151617181920");
     if (!code.empty())
     {
         // Deploy the code on some fake account to be called later.
         Account account(0, 0);
-        account.setCode(bytes{code});
+        auto const latestVersion = se->evmSchedule(envInfo.number()).version;
```

Comment:
Can you run this change in separate PR?

Reply:
Not sure what you mean, `setCode` now has two parameters, so we should pass something anyway to compile it.

Reply:
I misread it. I thought it takes the latest schedule by default.

Reply:
It takes the schedule according to `--network` CLI argument (though there's no Istanbul yet, that would go into separate PR)